front/components/use/MainTab.tsx

```diff
@@ -0,0 +1,173 @@
+import { Menu } from "@headlessui/react";
+import {
+  BellAlertIcon,
+  MagnifyingGlassCircleIcon,
+  PencilSquareIcon,
+} from "@heroicons/react/24/outline";
+import { ChevronDownIcon, CodeBracketIcon } from "@heroicons/react/24/solid";
+import Link from "next/link";
+
+import { classNames } from "@app/lib/utils";
+import { WorkspaceType } from "@app/types/user";
+
+export default function MainTab({
+  currentTab,
+  owner,
+}: {
+  currentTab: string;
+  owner: WorkspaceType;
+}) {
+  const tabs = [
+    {
+      name: "Chat",
+      href: `/w/${owner.sId}/u/chat`,
+      icon: (
+        <CodeBracketIcon
+          className="mr-2 mt-0.5 h-4 w-4 flex-shrink-0"
+          aria-hidden="true"
+        />
+      ),
+    },
+    {
+      name: "Answers",
+      href: `/w/${owner.sId}/u/answers`,
+      icon: (
+        <MagnifyingGlassCircleIcon
+          className="mr-2 mt-0.5 h-4 w-4 flex-shrink-0"
+          aria-hidden="true"
+        />
+      ),
+    },
+    {
+      name: "Notes",
+      href: `/w/${owner.sId}/u/notes`,
+      icon: (
+        <PencilSquareIcon
+          className="mr-2 mt-0.5 h-4 w-4 flex-shrink-0"
+          aria-hidden="true"
+        />
+      ),
+    },
+    {
+      name: "Alerts",
+      href: `/w/${owner.sId}/u/alerts`,
+      icon: (
+        <BellAlertIcon
+          className="mr-2 mt-0.5 h-4 w-4 flex-shrink-0"
+          aria-hidden="true"
+        />
+      ),
+    },
+  ] as { name: string; href: string; icon: JSX.Element }[];
```

Comment:
nit: why `as {}[]` instead of `const tabs: {}[] =` ? The latter actually type-checks

Reply:
me being a bad typescripter will address and merge :+1: